### DISCUSSION TITLE
Build with multi thread in multi i18n locales

### DISCUSSION THREAD

**Discussion by 2heal1** [facebook/docusaurus#5620] [Q&A] [answered]
I have `zh-cn`、`en` locales，and when i build ,it will take 6 minutes .
It takes too much time.
Can it use thread-loader or other method to speedup production build?

**Comment by slorber** [comment, marked as answer]
II need to write the doc for this but you can cache `node_modules/.cache` in your production CI across build, this will leverage Webpack 5 persistent caching and next builds will be faster.

You can also try using esbuild like we do on Docusaurus website, see https://github.com/facebook/docusaurus/issues/4765

Never tried thread-loader, that's worth taking a look at but I've other things to work on before investing again on build perfs

**Reply by 2heal1** [reply]
thanks ~
